chapters/hub/hub-api.md

```diff
@@ -0,0 +1,67 @@
+# Hub API
+
+Jina Hub exposes several handy commands, namely `list`, `build` and `push` with configurable parameters. These APIs constitute the user’s experience using `jina cli` for a seamless experience with the Hub.
```

Comment:
Can you provide a short summary of this guide in two sentences? 

Something like: Jina Hub API allows users to do x,y,z. This guide covers installing and setting up the API.

Reply:
Maybe we can link those terms to the summary? in rst is just with :term: ` Jina Hub ` and then we have the definition there?

Reply:
I've updated the summary and linked the terms with `::`, confirming on the preview